src/themes/default/components/core/blocks/Footer/Footer.vue

```diff
@@ -15,9 +15,12 @@
                 {{ $t('Orders') }}
               </h3>
               <div class="mt15">
-                <router-link class="cl-secondary" :to="localizedRoute('/my-account')" exact>
+                <router-link v-if="isLogged" class="cl-secondary" :to="localizedRoute('/my-account')" exact>
```

Comment:
isLogged is checked only on client-side, so it could potentially cause a hydration error. Safer to use v-show in that case
```suggestion
                <router-link v-show="isLogged" class="cl-secondary" :to="localizedRoute('/my-account')" exact>
```